File: csharp/raindrops/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        string retVal= "";
        if (number % 3 == 0)
            retVal += "Pling";
        if (number % 5 == 0)
            retVal += "Plang";
        if (number % 7 == 0)
            retVal += "Plong";
        if (retVal == "") 
            retVal = number.ToString();
        return retVal;
    }
}

File: python/raindrops/raindrops.py
def convert(number):
    retVal = ""
    if (number % 3 == 0):
        retVal += "Pling"
    if (number % 5 == 0):
        retVal += "Plang"
    if (number % 7 == 0):
        retVal += "Plong"
    if (number % 3 != 0 and number % 5 != 0 and number % 7 != 0):
        retVal = str(number)
    return retVal
